### PR TITLE
Add RDoc links for delegated methods

### PR DESCRIPTION
**Pull request by BurdetteLamar**
There is a list of delegated methods in RDoc for CSV.open.  Each now is a link to its documentation, either here in CSV or in class IO.

### CODE REVIEW

File: lib/csv.rb [hunk 1]
```diff
@@ -989,40 +989,41 @@ def generate_line(row, **options)
...
-    # * seek()
-    # * stat()
-    # * sync()
-    # * sync=()
-    # * tell()
-    # * to_i()
-    # * to_io()
-    # * truncate()
-    # * tty?()
+    # For convenience, an open CSV object will delegate to many methods in class IO.
```
Comment (kou) [inline lib/csv.rb:992]:
```suggestion
    # For convenience, an opened CSV object will delegate to many methods in class IO.
```

Comment (BurdetteLamar) [reply lib/csv.rb]: Done.

**Comment by BurdetteLamar** [comment]
@kou, I will not merge this or any PR;  leaving that for you.

**Comment by kou** [comment]
Thanks.

**Pull request merged**